proto/coprocessor.proto

```diff
@@ -21,7 +21,9 @@ message Request {
     kvrpcpb.Context context = 1;
     int64 tp = 2;
     bytes data = 3;
-    repeated KeyRange ranges = 4;
+    bool enable_match = 4;
```

Comment:
this will break the compatibility, please use larger number for new added fields.